api.ts

```diff
@@ -12095,13 +12923,60 @@ export const PlaidApiAxiosParamCreator = function (configuration?: Configuration
             };
         },
         /**
-         * `/income/verification/documents/download` provides the ability to download the source paystub PDF that the end user uploaded via Paystub Import.  The response to `/income/verification/documents/download` is a ZIP file in binary data. The `request_id`  is returned in the `Plaid-Request-ID` header.  In the future, when Digital Verification is available, the most recent file available for download with the payroll provider will also be available from this endpoint.
+         * `/income/verification/create` begins the income verification process by returning an `income_verification_id`. You can then provide the `income_verification_id` to `/link/token/create` under the `income_verification` parameter in order to create a Link instance that will prompt the user to go through the income verification flow. Plaid will fire an `INCOME` webhook once the user completes the Payroll Income flow, or when the uploaded documents in the Document Income flow have finished processing. 
+         * @summary Create an income verification instance
+         * @param {IncomeVerificationCreateRequest} incomeVerificationCreateRequest 
+         * @param {*} [options] Override http request option.
+         * @throws {RequiredError}
+         */
+        incomeVerificationCreate: async (incomeVerificationCreateRequest: IncomeVerificationCreateRequest, options: any = {}): Promise<RequestArgs> => {
```

Comment:
This function was  added but the `postIncomeVerificationCreate` was removed. This looks ok but would love confirmation here

Reply:
I think this is expected. the naming `postIncomeVerificationCreate` I think was incorrect in the first place